test/exitHandler.js

```diff
@@ -303,6 +330,59 @@ contract('ExitHandler', (accounts) => {
         
       });
 
+      it('Should allow to challenge limbo exit and prevent it', async () => {
+        // utxo that will have spend exit utxo
+        const period = await submitNewPeriod(depositTx);
+        const spendTx = Tx.transfer(
+          [new Input(new Outpoint(transferTx.hash(), 0))],
+          [new Output(50, charlie)]
+        ).sign([bobPriv]);
+
+        const depositProof = period.proof(depositTx);
+        const outputIndex = 1;
+        const exitStake = exitHandler.exitStake();
+        const signature = (transferTx + c).sign(alicePriv);
```

Comment:
`A sends B a signature on the hash of the in-flight transaction plus some special constant. `
Need to fix it